topics/metrics/README.md

```diff
@@ -33,8 +33,8 @@ host. This can occur when too many requests are in-flight for a single host.
 Connection timeouts occur when the process of establishing new connections is
 unresponsive (default: 5 seconds).
 
-[`cass_session_get_metrics()`]: http://datastax.github.io/cpp-driver/api/struct.CassSession/#1ab3773670c98c00290bad48a6df0f9eae
-[`CassMetrics`]: http://datastax.github.io/cpp-driver/api/struct.CassMetrics/
-[`requests`]: http://datastax.github.io/cpp-driver/api/struct.CassMetrics/#attribute-requests
-[`stats`]: http://datastax.github.io/cpp-driver/api/struct.CassMetrics/#attribute-stats
-[`errors`]: http://datastax.github.io/cpp-driver/api/struct.CassMetrics/#attribute-errors
+[`cass_session_get_metrics()`]: http://docs.datastax.com/en/developer/cpp-driver/latest/api/struct.CassSession/#1ab3773670c98c00290bad48a6df0f9eae
```

Comment:
Should be `http://docs.datastax.com/en/developer/cpp-driver/latest/api/struct.CassSession/#function-cass_session_get_metrics`